### PR TITLE
task01-commit01

### PR DESCRIPTION
**Pull request by AlexDovgalenko**
tezt

tetef d
f
s
d sgs

### CODE REVIEW

File: src/task1.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+task 01
+task 01
+task 01
+task 01
```
Comment (AlexDovgalenko) [inline src/task1.txt:4]: not sure about this line

Comment (azhelezny) [reply src/task1.txt:4]: it's OK

Comment (azhelezny) [inline src/task1.txt:4]: this is wrong

**Review by azhelezny — CHANGES_REQUESTED** [review]
fix my comments

File: src/task1.txt (hunk 1 above)
Comment (AlexDovgalenko) [reply src/task1.txt:4]: ok, let me fix it

**Review by azhelezny — APPROVED** [review]
well done

**Pull request merged**